--- a/config.h	2020-03-29 04:32:19.852986702 +0200
+++ b/config.h	2020-03-29 10:34:42.183320855 +0200
@@ -86,24 +86,24 @@ unsigned int tabspaces = 8;
 static const char *colorname[] = {

   /* 8 normal colors */
-  [0] = "#282828", /* hard contrast: #1d2021 / soft contrast: #32302f */
+  [0] = "#fbf1c7", /* hard contrast: #f9f5d7 / soft contrast: #f2e5bc */
   [1] = "#cc241d", /* red     */
   [2] = "#98971a", /* green   */
   [3] = "#d79921", /* yellow  */
   [4] = "#458588", /* blue    */
   [5] = "#b16286", /* magenta */
   [6] = "#689d6a", /* cyan    */
-  [7] = "#a89984", /* white   */
+  [7] = "#7c6f64", /* white   */

   /* 8 bright colors */
   [8]  = "#928374", /* black   */
-  [9]  = "#fb4934", /* red     */
-  [10] = "#b8bb26", /* green   */
-  [11] = "#fabd2f", /* yellow  */
-  [12] = "#83a598", /* blue    */
-  [13] = "#d3869b", /* magenta */
-  [14] = "#8ec07c", /* cyan    */
-  [15] = "#ebdbb2", /* white   */
+  [9]  = "#9d0006", /* red     */
+  [10] = "#79740e", /* green   */
+  [11] = "#b57614", /* yellow  */
+  [12] = "#076678", /* blue    */
+  [13] = "#8f3f71", /* magenta */
+  [14] = "#427b58", /* cyan    */
+  [15] = "#3c3836", /* white   */
 };

 /*
